osu.Game.Rulesets.Mania/Beatmaps/StageDefinition.cs

```diff
@@ -20,6 +21,18 @@ public struct StageDefinition
         /// </summary>
         /// <param name="column">The 0-based column index.</param>
         /// <returns>Whether the column is a special column.</returns>
-        public bool IsSpecialColumn(int column) => Columns % 2 == 1 && column == Columns / 2;
+        public bool IsSpecialColumn(int column)
+            // TODO: Inline this method.
+            => Columns % 2 == 1 && column == Columns / 2;
+
+        public ColumnType TypeOfColumn(int column)
```

Comment:
Tests can be added to test this method, but as the original code is not covered by tests...